src/Centreon/Domain/PlatformTopology/PlatformTopologyService.php

```diff
@@ -109,7 +109,11 @@ public function addPlatformToTopology(PlatformTopology $platformTopology): void
             $this->setMonitoringServerId($platformTopology, $isLocalhost);
         }
 
-        $this->checkForAlreadyRegisteredSameNameOrAddress($platformTopology);
+
+        if (PlatformTopology::TYPE_MBI !== $platformTopology->getType()) {
```

Comment:
`PlatformTopology::TYPE_MAP` should not be handled too ?

Reply:
MBI type is the only one allowed to have same address as its parent (eg: address of the remote its linked too).
MAP needs to follow the same rules as the others and needs to have an unique name and address